SimPEG/EM/TDEM/ProblemTDEM.py

```diff
@@ -719,7 +878,62 @@ def getRHSDeriv(self, tInd, src, v, adjoint=False):
         # right now, we are assuming that s_e, s_m do not depend on the model.
         return Utils.Zero()
 
+    def getInitialFields(self):
```

Comment:
Similar story here, do you think we can easily generalize the current `getInitialFields` to accommodate this case?